### PR TITLE
RTL fix

### PR DESCRIPTION
**Pull request by Danoosh**
I've not been tested this change. Please check it and merge if everything is OK. @AlexEven

### CODE REVIEW

**Comment by AlexEven** [comment]
Thank @Danoosh for this PR!
Can you please explain why replacing locale by iso would change anything for RTL?
I'll see with the teach team as it's always tricky to change the email generator :)

Thanks!

**Comment by AlexEven** [comment]
@eternoendless Can you please have a look?

**Comment by Danoosh** [comment]
@AlexEven I've used language code (without country code) before in this line:

https://github.com/iPresta/ps_emailgenerator/blob/f96d0fa50fbc81733e6ade1e3d72ee00dcace22a/ps_emailgenerator.php#L27

We nedd to check if iso code is exists in defined array or not.

**Review by eternoendless — APPROVED** [review]

File: ps_emailgenerator.php [hunk 1]
```diff
@@ -231,11 +231,12 @@ public function generateEmail($template, $locale)
...
         $LOCALE = $locale;
+        $iso = preg_replace('/-.*/','',$locale);
 
         $emailPublicWebRoot = Tools::getShopDomain(true).__PS_BASE_URI__.'modules/ps_emailgenerator/templates/';
-        $emailLangIsRTL = in_array($locale, self::$_rtl_langs); // see header.php
+        $emailLangIsRTL = in_array($iso, self::$_rtl_langs); // see header.php
         $emailDefaultFont = '';
-        if (array_key_exists($locale, self::$_lang_default_font)) {
+        if (array_key_exists($iso, self::$_lang_default_font)) {
             $emailDefaultFont = (self::$_lang_default_font[$locale]).',';
```
Comment (eternoendless) [inline ps_emailgenerator.php]: You forgot to change `$locale` to `$iso` here

**Review by eternoendless — CHANGES_REQUESTED** [review]

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @Danoosh

**Pull request merged**